internal/api/gc_coordinator.go

```
@@ -0,0 +1,50 @@
+
+import (
+	"log"
+	"runtime"
+	"sync"
+	"time"
+)
+
+// postTestGC coalesces cleanup requests from concurrently finishing test
+// batches. The request is made only after all workers in that batch have
+// exited, so GC is a final cleanup step rather than a per-node throttle.
+var postTestGC struct {
```

golangci-lint (2.12.2)

[error] postTestGC is a global variable

(gochecknoglobals)